docs/gl_objects/users.rst

```diff
@@ -106,6 +106,12 @@ List a user's starred projects
 
     user.starred_projects.list()
 
+If the GitLab instance has new user account approval enabled some users may
+have ``user.state == 'blocked_pending_approval'``. Administrators can approve
+and reject such users::
+
+    user.approve()
```

Comment:
How do we "reject" someone?

Reply:
I've added a mention of `reject()` as well.